src/DonationForms/resources/registrars/templates/groups/BillingAddress.tsx

```diff
@@ -185,7 +187,7 @@ function StateFieldContainer({
                 ) : (
                     <input
                         type="text"
-                        value={''}
+                        value={state}
```

Comment:
I got this warning in the console. What are your thoughts on removing the value prop of the input all together to keep it uncontrolled? Also above this line is a conditional text input with a value of `loading..` I don't see an attempt to load values here. If we remove it as well the console warnings no longer display, states still can load when available and it functions as a text input when needed.

<img width="1308" alt="Screen Shot 2024-01-02 at 12 08 46 PM" src="https://github.com/impress-org/givewp/assets/75056371/8aee32ed-0aa2-44e4-805e-e984ec62abca">

Reply:
Good catch @JoshuaHungDinh! I simplified that on 41310a61dc4b5135381674d3d602ae1f786e8b94.

The _Loading..._ text is displayed while checking/loading options based on the current country selection. In some cases, that input is replaced with a pre-populated dropdown. Before the commit above, since we were setting a value for that _Loading..._ text, we had to set something else later. I moved that text to be a placeholder in the field instead so that problem does not happen.

Reply:
Great idea!

Everything seems to be working smoothly. I just noticed that we can remove line 69 (unused variable). Could you double-check that, and once confirmed, this will be all set for QA.